src/main/java/com/iota/iri/network/NeighborRouterImpl.java

```diff
@@ -839,6 +843,10 @@ private static boolean isURIValid(final URI uri) {
             log.error("'{}' is not a valid URI schema, only TCP ({}) is supported", uri, PROTOCOL_PREFIX);
```

Comment:
The docs say its invalid when null, however we dont actually check for this. It was already like this, but can you add it anyways?

Reply:
nice catch